tests/unit/plugins/lookup/test_onepassword.py

```diff
@@ -265,6 +265,23 @@ def test_op_lookup(mocker, cli_class, vault, queries, kwargs, output, expected):
     assert result == expected
 
 
+@pytest.mark.parametrize(
+    ("vault", "queries", "kwargs", "output", "expected"),
+    (
+        (item["vault_name"], item["queries"], item.get("kwargs", {}), item["output"], item["expected"])
+        for item in SSH_KEY_MOCK_ENTRIES
+    )
+)
+def test_ssh_key(mocker, vault, queries, kwargs, output, expected):
```

Comment:
Can you put this test into a new file `test_onepassword_ssh_key.py`? This test will not be run if someone in the future modifies `plugins/lookup/onepassword_ssh_key.py` because it's in the wrong file (for ansible-test's change detection).